.gitignore

```diff
@@ -1,4 +1,7 @@
+.idea/**
 /bin/
 junit.xml
 /reports/*
 /config/prometheus/config.yaml
+loki-objectstorage-secret.yaml
+observatorium-logs-template.yaml
```

Comment:
You can make this part of your personal $home git ignore file. We should not clutter the project ones with personal work habits.

Reply:
ok ... removed that one ... agree